Resources/Catalog/FileCatalogClient.py

```diff
@@ -38,10 +46,18 @@ def isOK( self, timeout = 120 ):
         self.available = True
     return S_OK( self.available )
 
+  def hasCatalogMethod( self, methodName ):
```

Comment:
I still don't understand why not using hasattr ? This will work transparently in most of the classes, except the LCGCombinedClient, in which you can implement __hasattr__